testplan/runnable/base.py

```diff
@@ -1250,6 +1250,11 @@ def _invoke_exporters(self) -> None:
         # Add this logic into a ReportExporter(Runnable)
         # that will return a result containing errors
 
+        if isinstance(self._result.test_report, TestReport) and (
```

Comment:
Add an is_empty method to some report base class? And check both entries and logs (maybe something else too but I cannot think of right now)